internal/services/automation/registration.go

```diff
@@ -28,6 +28,7 @@ func (r Registration) Resources() []sdk.Resource {
 		SourceControlResource{},
 		WatcherResource{},
 		Python3PackageResource{},
+		AutomationPowerShell72ModuleResource{},
```

Comment:
Since this file only contains automation related resources/data sources I think it's fine to shorten this
```suggestion
		PowerShell72ModuleResource{},
```

Reply:
Done ✅